### PR TITLE
Docs demo 1

### PR DESCRIPTION
**Pull request by corinnebosley**
This is a notebook demonstrating how to link up some aspects of a work flow, but it has a few small issues I would like some advice on.

I am not happy that I have got my own data path in there instead of iris.sample_data_path, but I am not sure how to get that to work.

I think there could be a little too much comment text in there (although I'm not sure, I think it might also be quite useful, can't seem to find the balance).

There is some overlap between this notebook and the one I just merged by @marqh, I am not sure how much is necessary or useful.

Thanks

### CODE REVIEW

File: doc/developingCubeBrowsers/interactions_2.ipynb [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "ipynb = IPython.get_ipython()\n",
```
Comment (marqh) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: you get this from import `cube_browser`

File: doc/developingCubeBrowsers/interactions_2.ipynb [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This filepath is a directory containing the set of options for the first selection box.\n",
+    "path = '/home/h06/cbosley/Projects/iris-sample-data/sample_data'\n",
```
Comment (marqh) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: `path = iris.sample_data_path()`
?

**Comment by marqh** [comment]
> I am not happy that I have got my own data path in there instead of iris.sample_data_path, but I am not sure how to get that to work.

```
import iris
spath = iris.sample_data_path()

```

File: doc/developingCubeBrowsers/interactions_2.ipynb (hunk 2 above)
Comment (corinnebosley) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: I had trouble getting this to work, it kept telling me it required one argument.

File: doc/developingCubeBrowsers/interactions_2.ipynb [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.11"
+  },
+  "widgets": {
```
Comment (marqh) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: can you remove all of these from the notebook before you save it, to reduce our source overhead?

File: doc/developingCubeBrowsers/interactions_2.ipynb (hunk 1 above)
Comment (corinnebosley) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: So I can take out this whole cell?

File: doc/developingCubeBrowsers/interactions_2.ipynb (hunk 2 above)
Comment (corinnebosley) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: I tried this:
```python
path = iris.sample_data_path('*')
```
but it just gave me the GloSea4 files then.

**Pull request closed without merging**